src/frontend/dbusfrontend/dbusfrontend.cpp

```diff
@@ -23,6 +23,17 @@
 
 namespace fcitx {
 
+namespace {
+
+void buildFormattedTextVector(
```

Comment:
two choice about the naming:
1. just construct and return the vector
2. rename from build to append (if same vector is used by this function twice).